crates/uv/tests/it/edit.rs

```diff
@@ -2187,7 +2218,7 @@ fn add_workspace_editable() -> Result<()> {
 
     let child1 = context.temp_dir.join("child1");
     let mut add_cmd = context.add();
-    add_cmd.arg("child2").arg("--editable").current_dir(&child1);
+    add_cmd.arg("child2").current_dir(&child1);
```

Comment:
This is a workspace member, passing `--editable` is just ignored. Should we _not_ error here?

Reply:
I think if an option is redundant then it makes sense not to error, right?

This change is because we error here now?

Reply:
Yeah. I mean it's a bit weird. `--editable` and `--no-editable` are flags on `uv sync` for workspaces, but they have no effect here. It's not a persistent setting.

Reply:
We could error if they pass `--no-editable`, because it won't have the intended effect.

Reply:
Can't workspace members have `editable = true/false` in their source definition?

Reply:
Nevermind, that's not the case. Yeah I'd expect an error on `--no-editable`.